### PR TITLE
Allow voting in democracy

### PR DESCRIPTION
**Pull request by gavofyork**
This doesn't let accounts submit referenda yet, but does let accounts vote and set delegates on refernda already underway.

### CODE REVIEW

File: runtime/polkadot/src/lib.rs [hunk 1]
```diff
@@ -110,7 +110,12 @@ pub struct BaseFilter;
 impl Filter<Call> for BaseFilter {
 	fn filter(call: &Call) -> bool {
 		match call {
-			Call::Parachains(parachains::Call::set_heads(..)) => true,
+			Call::Parachains(parachains::Call::set_heads(..))
+				| Call::Democracy(democracy::Call::vote(..))
```
Comment (bkchr) [inline runtime/polkadot/src/lib.rs:114]: Maybe we should move the allowed calls from the bottom to the top here as well?

**Comment by shawntabrizi** [comment]
How will a proposal get in the queue to be voted on?

File: runtime/polkadot/src/lib.rs (hunk 1 above)
Comment (gavofyork) [reply runtime/polkadot/src/lib.rs:114]: It wouldn't gain much as there's usage of defaults for selection happening both ways (e.g. `Call::Vesting(vesting::Call::vested_transfer(..)) | Call::Indices(indices::Call::transfer(..))`)

File: runtime/polkadot/src/lib.rs [hunk 2]
```diff
@@ -394,20 +399,38 @@ impl democracy::Trait for Runtime {
 	type VotingPeriod = VotingPeriod;
 	type MinimumDeposit = MinimumDeposit;
 	/// A straight majority of the council can decide what their next motion is.
-	type ExternalOrigin = collective::EnsureProportionAtLeast<_1, _2, AccountId, CouncilCollective>;
+	type ExternalOrigin = system::EnsureOneOf<AccountId,
```
Comment (joepetrowski) [inline runtime/polkadot/src/lib.rs:402]: Dumb question, but would this `AccountId` allow any account to take on this origin? The following two lines make sense to me.

Comment (gavofyork) [reply runtime/polkadot/src/lib.rs:402]: no - it's an overhang from rust's less-than-ideal type inference; we need to provide it explicitly, but it's not one of the origins.

**Comment by sheenhx** [comment]
is there plan to let people to submit proposals? If yes what is the number of the ticket?

**Comment by joepetrowski** [comment]
> is there plan to let people to submit proposals? If yes what is the number of the ticket?

Yes, should be open in the next few weeks. There is no ticket number, it's just part of the Polkadot launch plan.
https://polkadot.network/explaining-the-polkadot-launch-process/

**Pull request merged**